examples/llama2-7b/base-model.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: substratus.ai/v1
```

Comment:
Adding a comment to the top of the file to drive the point home
```suggestion
# ensure this manifest is rendered via envsubst and $HUGGINGFACE_TOKEN  is set prior to apply

apiVersion: substratus.ai/v1
```

Reply:
Or feel free to use the same format/position as the comment in the 70b manifest: https://github.com/substratusai/substratus/pull/146/files#diff-c15d5042193b36d46847eba0cac2a61c593da60e386243bc5ae2aaf60a9c3acaR9-R12

Reply:
I rather keep it next to where HUGGINGFACE_TOKEN is used. Or you mean to have 2 comments? 1 at the top and 1 next to where the variable is used?

Reply:
The reason being is some people might just want to copy paste directly in the file instead of relying on `envsubst`. I just gave that as an example on how you could replace it.